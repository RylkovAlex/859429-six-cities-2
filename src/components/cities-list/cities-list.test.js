import React from 'react';
import renderer from 'react-test-renderer';
import CitiesList from './cities-list';

it(`OfferPage correctly renders after relaunch`, () => {
  const props = {
    city: {
      name: `Amsterdam`,
      location: {
        latitude: 52.370216,
        longitude: 4.895168,
        zoom: 10
      }
    },
    cities: [
      {
        name: `1`,
        location: {
          latitude: 1,
          longitude: 2,
          zoom: 3
        }
      },
      {
        name: `2`,
        location: {
          latitude: 1,
          longitude: 2,
          zoom: 3
        }
      }
    ],
    cityClickHandler: jest.fn(),
  };

  const tree = renderer
    .create(<CitiesList {...props}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
