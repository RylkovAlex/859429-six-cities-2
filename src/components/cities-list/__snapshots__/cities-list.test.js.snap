// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferPage correctly renders after relaunch 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span
        onClick={[MockFunction]}
      >
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span
        onClick={[MockFunction]}
      >
        Cologne
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item "
      href="#"
    >
      <span
        onClick={[MockFunction]}
      >
        Paris
      </span>
    </a>
  </li>
</ul>
`;
